frontend/src/types/waiverTypes.tsx

```diff
@@ -17,6 +17,7 @@ export interface LoadedWaiver extends WaiverReducerDispatchBase {
 }
 export interface ClickOptionalClause extends WaiverReducerDispatchBase {
   type: WaiverActions;
+  agreed: boolean;
```

Comment:
I feel hesitant to move the types in waiverTypes.tsx into AdminTypes file since the type in waiverTypes.tsx are specific to the waiverReducer which is specific only to the waiver registration. And Yep, let me rename the file.